src/util-napatech.c

```diff
@@ -800,10 +800,21 @@ static uint32_t CountWorkerThreads(void)
 
                         char copystr[16];
                         strlcpy(copystr, lnode->val, 16);
-
-                        ByteExtractStringUint8(&start, 10,  0, copystr);
-                        ByteExtractStringUint8(&end, 10,  0, strchr(copystr, '-') + 1);
-
+                        if (StringParseUint8(&start, 10, 0, (const char *)copystr) < 0) {
+                            SCLogError(SC_ERR_INVALID_VALUE, "Napatech invalid"
```

Comment:
Please use `FatalError` to replace the `SCLogError(..); exit(EXIT_FAILURE);` pattern.

Reply:
Changing for this one. Rest will be fixed later by https://redmine.openinfosecfoundation.org/issues/3188